requirements/common.txt

```diff
@@ -408,42 +408,44 @@ django-cache-memoize==0.1.6 \
-jx-mysql==3.45.20031 \
-    --hash=sha256:2bcffa2e013126ec3267449d3119fcff017f4c6c7d38a04fbc6194b81cda52dd
-mo-dots==3.46.20032 \
-    --hash=sha256:698380ed29c0be3677b8e68ae2e74cadac37299bf6914aefb695c2b34d14bf3f
+jx-bigquery==3.48.20042 \
+    --hash=sha256:c9e22d07fbf3cb5032ecaefa849ef422e2b5c7f87615bdfd3dd13b0b834d94a2
+jx-mysql==3.49.20042 \
+    --hash=sha256:adb47e8e13151178cbb81e3e352776ec848b799e9b038de7796a889782564519
+jx-python==3.47.20042 \
+    --hash=sha256:6e51e112bcd9c9dbffb9f6f92c670e8f8af954c9cfafae8e79b6126c156b5dde
+jx-sqlite==3.42.20031 \
+    --hash=sha256:929070fd5b6663247bfa73254732f71eda817e30f3a25a04af4274f1bb156e04
+
+mo-collections==3.46.20032 \
+    --hash=sha256:b96df523a7894e78c9160b45426b8db45d47841f5185f4dd1ccfe9a3708e89d6
+mo-dots==3.47.20042 \
+    --hash=sha256:4bed834b4932a6b258d7d6244fed0931c4f2b1ae77b991302c4cf74515f47ff8
 mo-files==3.41.20031 \
     --hash=sha256:b5e7bd0b51844a54f5d9f6d8c8ef5b2c86defd81c49b2b5aed7850732042f7b3
+mo-future==3.47.20042 \
+    --hash=sha256:6b17166d41bd26d9204a7becdceb3f024f83095bf8caac979de163a2ff86ae34
+mo-json==3.47.20042 \
+    --hash=sha256:2710c8502e5c321bff18ae2eec33f1462b37c02e42c9998ffac0b6068849c512
+mo-json-config==3.42.20031 \
+    --hash=sha256:65f926de86c4cd21f6b13e4dbb858927a0e8be5e83635daa64ffe9f5133eb36b
+mo-kwargs==3.47.20042 \
+    --hash=sha256:2b6c92f55c88a2320057f9fc774b8b27ff6ba8bf9e665b98b487dfc40a29a405
+mo-logs==3.47.20042 \
+    --hash=sha256:fc9afa0f225085a4db267d13f1bc5ebd1bdd0f0cb836c432002271d90cf91d89
 mo-math==3.41.20031 \
     --hash=sha256:76ac7894c45986fcbe84eaa638643561bae59ada8f523ed4e064c556fdd1f9d3
-jx-python==3.43.20031 \
-    --hash=sha256:85315b458954629b75902ca97841b0a0a57cd22d12ba2b85484b32604a41ad31
-mo-logs==3.46.20032 \
-    --hash=sha256:f69fa8aae7c3c493823259b7d28220b318bc5433b6bba53796824f7c7ba88c3d
-mo-sql==3.38.20031 \
-    --hash=sha256:18db8a496dc86a057cc8e6dd515369a23cd5113e049788744513ec88deee99ae
-mo-json==3.38.20029 \
-    --hash=sha256:c9fc794eca55bb220a050b0c3e5b34a807fbe3d7d72b84f5d76c1a32ef618600
-mo-kwargs==3.38.20029 \
-    --hash=sha256:f4075da450d9c73655c6431136c970648c4f57d1e29930abba006db83f2a5836
-mo-collections==3.38.20029 \
-    --hash=sha256:291069c12c7275fbbddc4437b961805a55ca501555c2dac6538f9d899feb0253
+mo-sql==3.47.20042 \
+    --hash=sha256:a6a93be00fe9f7cc667e25fb1f6cf267003b457a0cce8226edcfb46a58f291cf
 mo-testing==3.38.20029 \
     --hash=sha256:48c4a9b10f2d20a6fdfe66a6e72d9c47c456023eefcc3845ed6c988b44380cac
+mo-times==3.46.20032 \
+    --hash=sha256:94661e2fdfe43b85a1bdd3a40e2b77c4aa91c5863389acb0c2017e4690415af4
 mo-threads==3.42.20031 \
     --hash=sha256:15a7caddbfd5067be737909c214861898a357370f41d7e590ced7061797019b3
-mo-json-config==3.42.20031 \
-    --hash=sha256:65f926de86c4cd21f6b13e4dbb858927a0e8be5e83635daa64ffe9f5133eb36b
-jx-sqlite==3.42.20031 \
-    --hash=sha256:929070fd5b6663247bfa73254732f71eda817e30f3a25a04af4274f1bb156e04
+pyLibrary==3.47.20042 \
```

Comment:
What's happening here? Sorting of the libraries and newer versions?

Reply:
New versions, and sorting.